### ISSUE TITLE
Support social-auth-app-django 5.0.0

### ISSUE THREAD

**Issue by bj00rn** [sunscrapers/djoser#668]
Hi,

Any reason why we can't support `social-auth-app-django` 5?

I see that the dependency has been loosened in master.

https://github.com/python-social-auth/social-app-django/blob/master/CHANGELOG.md#500---2021-08-05

Thanks!

**Comment by bj00rn** [comment]
https://github.com/sunscrapers/djoser/issues/655

**Comment by bj00rn** [comment]
https://github.com/sunscrapers/djoser/issues/651

**Comment by skonikevrone** [comment]
@dekoza @tomwojcik

New release would solve django 4 compatibility. `social-auth-app-django` 5 has important bug fix.

**Comment by skonikevrone** [comment]
> > @dekoza @tomwojcik
> > New release would solve django 4 compatibility. `social-auth-app-django` 5 has important bug fix.
> 
> We're not the maintainers anymore.

Sorry to bother you.
Any chance you know someone whom we can ping?

**Comment by tomwojcik** [comment]
@skonikevrone No problem. @haxoza is your best bet.

**Comment by bj00rn** [comment]
@haxoza we get a new release please you think? 

Seems there are issues with latest commit on master.
Commit https://github.com/sunscrapers/djoser/commit/ede06f257950705bbf6ded47d3c6daf11eb36fe3 is working fine though

**Comment by sinisarudan** [comment]
Yes, This issue requires hacks for `social-auth-app-django` like the one I have introduced
https://github.com/python-social-auth/social-app-django/issues/386#issuecomment-1252114612
The contributor to the `social-auth-app-django` 
has answered:

> Use social-auth-app-django==5.0.0 instead for compatiblity with recent Django releases.

that we cannot use actually, while the djoser doesn't overcome this dependency requirement:

```
Cannot install ... social-auth-app-django==5.0.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested social-auth-app-django==5.0.0
    djoser 2.1.0 depends on social-auth-app-django<5.0.0 and >=4.0.0
```

**Comment by albjoaov** [comment]
Hello! Any updates on this?

**Comment by Geg3mon** [comment]
Hi, in your **settings.py**
```
import django
from django.utils.encoding import force_str
django.utils.encoding.force_text = force_str
```

**Comment by bj00rn** [comment]
@dekoza @tomwojcik

Any other guesses to to who maintains this repo? @haxoza is unresponsive it seems.

This is a showstopper for many who are using this library

Thanks!

**Comment by tomwojcik** [comment]
@bj00rn I will take over from now on, but I can spend only a few hours a month on this project. I will look over the issues soon. For now, the new release is the top prio.

**Comment by bj00rn** [comment]
> @bj00rn I will take over from now on, but I can spend only a few hours a month on this project. I will look over the issues soon. For now, the new release is the top prio.

Good news!
